documentation/milestones/individual_milestones/search_mvp.md

```diff
@@ -0,0 +1,210 @@
+
+## Goals
+
+### Business description & value
+<!-- Required -->
+
+The launch of a basic search feature primarily proves our ability to integrate front-end and backend systems in a production environment, while also providing a minimum viable product (MVP) that we can leverage for targeted user testing.
+
+As a result, this deliverable aims to:
+
+- Successfully integrate the front-end with the API for the first time in production
+- Create the internal infrastructure needed to support future search functionality
+- Establish simpler.grants.gov as the location where that functionality will be released
+- Deliver a minimum viable product (MVP) that we can use to collect user feedback from a targeted set of stakeholders
+
+By contrast, this MVP version of search ***does not*** attempt to:
+
+- Replicate *all* of the search functionality currently supported by grants.gov
+- Dramatically improve the accuracy of search results relative to those returned by grants.gov
+- Replace grants.gov as the primary platform applicants use to find funding opportunities
+
+### User Stories
+<!-- Required -->
+
+- As a **system-to-system user**, I want:
+  - To be able to access all of the search features via the API, so that search results are the same whether I'm using the API or the UI.
+  - The search functionality to be outlined in the API docs, so that I don't have to rely on experimentation to learn how to search for opportunities.
+- As a **prospective applicant**, I want:
+  - To be able to be able to search with filters applied, so that the only search results returned are ones that match both the keyword and field-based criteria I specified
+  - The search results to have a link to the opportunity on grants.gov, so that I don't have to repeat my search on grants.gov after finding an opportunity I want to apply for on simpler.grants.gov
+  - To receive an incentive if I participate in user research for simpler.grants.gov, so that I am fairly compensated for the time that I invest in testing out new features and answering questions
+  - Filters and search terms to be added to the url, so that I can save or share the link to a particular set of search results
+- As a **project maintainer**, I want:
+  - To be notified when simpler.grants.gov experiences downtime, so that I can quickly troubleshoot the issue with minimal impact on end users
+  - The system architecture we create for this deliverable to support future search features as well, so that we don't need to discard and re-build critical infrastructure
+
+## Technical description
+
+### Search API
+<!-- Optional -->
+
+The ability to search and filter opportunities should be provided through an API endpoint, which allows the user to:
+
+- Search the list of opportunities with open ended key words
+- Filter the search results by *at least* one structured field (e.g. award date, opportunity type, etc.)
+- Sort the search results based on the value of *at least*
```

Comment:
missing end of this sentence?

Reply:
Yup, lol! I finished the sentence in [this commit](https://github.com/HHS/simpler-grants-gov/pull/579/commits/0a05c4a125042018476cf6e64e3418b935b1fcee)